src/main/java/eod/card/concrete/normal/SupportAttack.java

```diff
@@ -73,15 +80,15 @@ public void onEventOccurred(GameObject sender, Event event) {
             if(event instanceof TargetedEvent) {
                 TargetedEvent e = (TargetedEvent) event;
                 if(e.getTarget() != holder) {
-                    ((Damageable) e.getAttacker()).damage(2);
+                    new Damage(new DamageParam(2), Effect.HandlerType.Rival).on((Damageable) e.getAttacker());
```

Comment:
Same as [this comment](https://github.com/Elegy-Composer/EOD-DM/pull/35#pullrequestreview-417917035)

Reply:
fixed